### PR TITLE
Folders organization, implement reducers, implement all components divided by containers

### PR DESCRIPTION
**Pull request by cochabambinoski**
Components created:
App
Book
BookList
BookForm

Reducers created based on these ones.

### CODE REVIEW

File: src/reducers/book.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * Implement the books.js reducer
+ * with CREATE_BOOK and REMOVE_BOOK actions.Remember,
+ *  your state can not be mutated!
+ */
+
+import { ADD_BOOK, REMOVE_BOOK } from '../actions/index';
+
+/* const {
+  ADD_BOOK
+} = require("../src/actions"); */
```
Comment (Oghenebrume50) [inline src/reducers/book.js]: Please delete commented codes, 
see

File: src/containers/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+
+function BooksList(properties) {
```
Comment (Oghenebrume50) [inline src/containers/BooksList.js]: Validate these properties with Proptypes

File: src/components/Book.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function Book(properties) {
```
Comment (Oghenebrume50) [inline src/components/Book.js]: Validate these properties with PropTypes

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @cochabambinoski @RoodzFernando good job working on this, this is coming along very well, please make changes as outlined below :+1:

PS: It seems you forgot to continue with gitflow on this milestone, since your PR is pointing to master, please change this to point to your development branch :+1: 

**Request for another review after making changes**

**RAPHAEL NORIODE** :mask: STAY_SAFE**

**Review by whiz25 — APPROVED** [review]
## Approved :clap: 
You have done the required changes and it's a wrap from me. Merge and continue to the next milestone.

### Martin Afani

**Pull request merged**